src/Logging/JsonFormatter.php

```diff
@@ -3,17 +3,32 @@
 namespace Laravel\Vapor\Logging;
 
 use Monolog\Formatter\JsonFormatter as BaseJsonFormatter;
+use Monolog\LogRecord;
 
 class JsonFormatter extends BaseJsonFormatter
 {
     /**
      * {@inheritdoc}
      */
-    public function format(array $record): string
+    public function format($record): string
     {
-        $record['context'] = array_merge(
-            $record['context'] ?? [], ['aws_request_id' => ($_ENV['AWS_REQUEST_ID'] ?? null)]
-        );
+        $context = ['aws_request_id' => ($_ENV['AWS_REQUEST_ID'] ?? null)];
+
+        if ($record instanceof LogRecord) {
```

Comment:
Under what circumstances can this be `LogRecord`?

Reply:
Monolog v3 ( aka Laravel 10 ).